app/controllers/places_controller.rb

```diff
@@ -1,21 +1,65 @@
 class PlacesController < ApplicationController
   
+  # GET ../places
   def index
-    @posts = policy_scope(Post)
+    @places = policy_scope(Place)
+    #@places = Place.all
+    #binding.pry
   end
 
+  # GET ../places/new
+  def new
+    @place = Place.new(owner_id: current_user.id)
+    skip_authorization
+  end
+
+  # POST ../places
+  def create
+    @place = current_user.owned_places.create(place_params)
+    skip_authorization
+    respond_to do |format|
+      if @place.save
+        format.html { redirect_to profile_path, notice: 'Place was successfully created.' }
+      else
+        format.html { render :new }
+      end
+    end
+  end
+  
+  #Created just for policy authorization purposes to use in the future
+  def edit
+    @place = Place.find(params[:id])
+    authorize @place
```

Comment:
Is this authorization working correctly?? I'm checking that the policy is based on the `user_id` attribute. It should be based in the `owner_id`, shouldn't it??